app/builders/immediate_email_builder.rb

```diff
@@ -28,6 +28,7 @@ def records
           subscriber_id: subscriber.id,
           created_at: now,
           updated_at: now,
+          content_id: content.instance_of?(ContentChange) ? content.content_id : nil,
```

Comment:
We could just do `content.try(:content_id)` here

Reply:
Oh yeah, that's a bit more future proof as well (maybe messages will have content ids in them at some point)